src/fsharp/FSharp.Core/local.fs

```diff
@@ -73,7 +73,7 @@ module internal List =
             cons
     
     let countBy (dict:Dictionary<_, int>) (keyf:'T -> 'Key) = 
-        let mutable ie = dict.GetEnumerator()
+        use mutable ie = dict.GetEnumerator()
```

Comment:
@asik that is a good point.  
It is true that the empty dispose implementation is an implementation detail.  But I wouldn't expect that change.

Kevin